crypto/crypto.ts

```diff
@@ -293,23 +295,6 @@ const stdCrypto: StdCrypto = ((x) => x)({
   },
 });
 
-const FNV_ALGORITHMS = ["FNV32", "FNV32A", "FNV64", "FNV64A"];
-
-/** Digest algorithms supported by WebCrypto. */
-const webCryptoDigestAlgorithms = [
-  "SHA-384",
-  "SHA-256",
-  "SHA-512",
-  // insecure (length-extendable and collidable):
-  "SHA-1",
-] as const;
-
-/** FNV (Fowler/Noll/Vo) algorithms names. */
-export type FNVAlgorithms = "FNV32" | "FNV32A" | "FNV64" | "FNV64A";
-
-/** Extended digest algorithm names. */
-export type DigestAlgorithmName = WasmDigestAlgorithm | FNVAlgorithms;
-
```

Comment:
I agree with these changes. There's no need to expose these types. However, these are also breaking changes. Let's first deprecate.

Reply:
I've reverted the externally-visible removals and renames and will address them in a follow-up PR.